q2cli/commands.py

```diff
@@ -180,7 +185,16 @@ def __call__(self, **kwargs):
         actions_module = importlib.import_module(module_path)
         action = getattr(actions_module, self.action['id'])
 
-        results = action(**arguments)
+        stdout = os.devnull
+        stderr = os.devnull
+        if verbose:
+            # `qiime.util.redirected_stdio` defaults to stdout/stderr when
```

Comment:
Another way to do this is to only invoke the context manager if not in verbose mode. This would avoid redirecting stdio when in verbose mode. On the fence, it probably doesn't matter much either way.